mmap/src/main/java/com/indeed/util/mmap/NativeBuffer.java

```diff
@@ -43,15 +42,10 @@
         }
         MMAP_THRESHOLD = mmapThreshold;
         try {
-            final Field theUnsafe = Unsafe.class.getDeclaredField("theUnsafe");
-            theUnsafe.setAccessible(true);
-            UNSAFE = (Unsafe) theUnsafe.get(null);
             FD_FIELD = FileDescriptor.class.getDeclaredField("fd");
             FD_FIELD.setAccessible(true);
         } catch (NoSuchFieldException e) {
             throw new RuntimeException(e);
-        } catch (IllegalAccessException e) {
```

Comment:
Are we sure it's safe not to catch this?

Reply:
It's only thrown by Field.get, which is removed in this patch. javac won't allow it to be caught if it's not thrown since it's checked.